src/context.rs

```diff
@@ -189,6 +189,8 @@ mod alloc_only {
         /// # }
         /// ```
         #[cfg_attr(not(feature = "rand-std"), allow(clippy::let_and_return, unused_mut))]
+        #[allow(unused_mut)] // ctx is not mutated under some feature combinations.
```

Comment:
I think you can put it on top of `let` statement to limit its scope.

Reply:
TIL, thanks man.